File: back-end/src/main/java/com/ta2khu75/quiz/entity/request/ExamRequest.java
package com.ta2khu75.quiz.entity.request;

import com.ta2khu75.quiz.entity.ExamLevel;
import com.ta2khu75.quiz.entity.ExamType;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;
@Data
public class ExamRequest {
	@NotBlank
	String title;
	@NotNull
	Integer time;
	@NotBlank
	String description;
	@NotNull
	ExamType examType;
	@NotNull
	ExamLevel examLevel;
}

File: front-end/src/model/request/ExamRequest.ts
import { AccessModifier } from "../AccessModifier";
import { ExamLevel } from "../ExamLevel";

export default interface ExamRequest  {
  title: string;
  duration: number;
  description: string;
  exam_level: ExamLevel;
  access_modifier: AccessModifier;
  exam_category_id: number;
}